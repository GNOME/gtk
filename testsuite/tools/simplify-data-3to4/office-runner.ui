<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <!-- interface-requires gtk+ 3.0 -->
  <object class="GtkWindow" id="window1">
    <property name="can_focus">False</property>
    <property name="border_width">12</property>
    <property name="title" translatable="yes">Office Runner</property>
    <property name="resizable">False</property>
    <property name="window_position">center</property>
    <child>
      <object class="GtkBox" id="box1">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="margin_left">12</property>
        <property name="margin_right">12</property>
        <property name="margin_bottom">12</property>
        <property name="orientation">vertical</property>
        <property name="spacing">12</property>
        <child>
          <object class="GtkNotebook" id="notebook1">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="show_tabs">False</property>
            <property name="show_border">False</property>
            <child>
              <object class="GtkGrid" id="grid1">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="row_spacing">12</property>
                <property name="column_spacing">12</property>
                <child>
                  <object class="GtkImage" id="run_image">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="vexpand">True</property>
                  </object>
                  <packing>
                    <property name="left_attach">0</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkLabel" id="label5">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="valign">start</property>
                    <property name="hexpand">True</property>
                    <property name="xalign">0</property>
                    <property name="label" translatable="yes"> • Click the "Run!" button, and then dash to your desk or meeting.
 • You are allowed to close your laptop lid to run faster
</property>
                    <property name="use_markup">True</property>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="top_attach">1</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkLabel" id="label7">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="label" translatable="yes">Rules</property>
                    <attributes>
                      <attribute name="scale" value="2"/>
                    </attributes>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
                <child>
                  <placeholder/>
                </child>
              </object>
            </child>
            <child type="tab">
              <object class="GtkLabel" id="label1">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="no">page 1</property>
              </object>
              <packing>
                <property name="tab_fill">False</property>
              </packing>
            </child>
            <child>
              <object class="GtkGrid" id="grid2">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="row_spacing">12</property>
                <property name="column_spacing">12</property>
                <child>
                  <object class="GtkImage" id="time_image">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="vexpand">True</property>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkLabel" id="time_label">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="hexpand">True</property>
                    <property name="vexpand">True</property>
                    <property name="label" translatable="no">00:00.00</property>
                    <attributes>
                      <attribute name="scale" value="10"/>
                      <attribute name="font-features" value="tnum 1"/>
                    </attributes>
                  </object>
                  <packing>
                    <property name="left_attach">0</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="position">1</property>
              </packing>
            </child>
            <child type="tab">
              <object class="GtkLabel" id="label2">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="no">page 2</property>
              </object>
              <packing>
                <property name="position">1</property>
                <property name="tab_fill">False</property>
              </packing>
            </child>
            <child>
              <object class="GtkGrid" id="grid3">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="hexpand">True</property>
                <property name="row_spacing">12</property>
                <property name="column_spacing">12</property>
                <child>
                  <object class="GtkImage" id="trophy_image">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                  </object>
                  <packing>
                    <property name="left_attach">0</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
                <child>
                  <object class="GtkBox" id="box2">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="valign">center</property>
                    <property name="orientation">vertical</property>
                    <child>
                      <object class="GtkLabel" id="result_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="valign">start</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="no">Silver Trophy!</property>
                        <property name="use_markup">True</property>
                        <style>
                          <class name="dim-label"/>
                        </style>
                        <attributes>
                          <attribute name="weight" value="bold"/>
                          <attribute name="scale" value="1.5"/>
                        </attributes>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">0</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="current_time_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="valign">start</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="no">You managed to finish the route with the 2nd best time ever, &lt;b&gt;48.15 seconds!&lt;/b&gt;	</property>
                        <property name="use_markup">True</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">1</property>
                      </packing>
                    </child>
                    <child>
                      <object class="GtkLabel" id="better_time_label">
                        <property name="visible">True</property>
                        <property name="can_focus">False</property>
                        <property name="valign">start</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="no">Only &lt;b&gt;3.75 seconds&lt;/b&gt; separate you from the gold trophy!</property>
                        <property name="use_markup">True</property>
                        <property name="wrap">True</property>
                      </object>
                      <packing>
                        <property name="expand">False</property>
                        <property name="fill">True</property>
                        <property name="position">2</property>
                      </packing>
                    </child>
                  </object>
                  <packing>
                    <property name="left_attach">1</property>
                    <property name="top_attach">0</property>
                    <property name="width">1</property>
                    <property name="height">1</property>
                  </packing>
                </child>
              </object>
              <packing>
                <property name="position">2</property>
              </packing>
            </child>
            <child type="tab">
              <object class="GtkLabel" id="label3">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="no">page 3</property>
              </object>
              <packing>
                <property name="position">2</property>
                <property name="tab_fill">False</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkButton" id="run_button">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="hexpand">True</property>
            <property name="vexpand">True</property>
            <property name="use_action_appearance">False</property>
            <child>
              <object class="GtkLabel" id="run_button_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="yes">Run!</property>
                <attributes>
                  <attribute name="scale" value="3"/>
                </attributes>
              </object>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">1</property>
          </packing>
        </child>
      </object>
    </child>
  </object>
</interface>
