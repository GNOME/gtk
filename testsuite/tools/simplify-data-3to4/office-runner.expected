<?xml version="1.0" encoding="UTF-8"?>
<interface>
  <object class="GtkWindow" id="window1">
    <property name="border_width">12</property>
    <property name="title" translatable="yes">Office Runner</property>
    <property name="resizable">0</property>
    <property name="window_position">center</property>
    <child>
      <object class="GtkBox" id="box1">
        <property name="margin_left">12</property>
        <property name="margin_right">12</property>
        <property name="margin_bottom">12</property>
        <property name="orientation">vertical</property>
        <property name="spacing">12</property>
        <child>
          <object class="GtkNotebook" id="notebook1">
            <property name="show_tabs">0</property>
            <property name="show_border">0</property>
            <child>
              <object class="GtkNotebookPage">
                <property name="child">
                  <object class="GtkGrid" id="grid1">
                    <property name="row_spacing">12</property>
                    <property name="column_spacing">12</property>
                    <child>
                      <object class="GtkImage" id="run_image">
                        <property name="vexpand">1</property>
                      </object>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label5">
                        <property name="valign">start</property>
                        <property name="hexpand">1</property>
                        <property name="xalign">0</property>
                        <property name="label" translatable="yes"> • Click the &quot;Run!&quot; button, and then dash to your desk or meeting.
 • You are allowed to close your laptop lid to run faster
</property>
                        <property name="use_markup">1</property>
                        <layout>
                          <property name="left_attach">1</property>
                          <property name="top_attach">1</property>
                        </layout>
                      </object>
                    </child>
                    <child>
                      <object class="GtkLabel" id="label7">
                        <property name="label" translatable="yes">Rules</property>
                        <attributes>
                          <attribute name="scale" value="2"></attribute>
                        </attributes>
                        <layout>
                          <property name="left_attach">1</property>
                        </layout>
                      </object>
                    </child>
                    <child>
                      <placeholder/>
                    </child>
                  </object>
                </property>
                <property name="tab">
                  <object class="GtkLabel" id="label1">
                    <property name="label" translatable="no">page 1</property>
                  </object>
                </property>
              </object>
            </child>
            <child>
              <object class="GtkNotebookPage">
                <property name="position">1</property>
                <property name="child">
                  <object class="GtkGrid" id="grid2">
                    <property name="row_spacing">12</property>
                    <property name="column_spacing">12</property>
                    <child>
                      <object class="GtkImage" id="time_image">
                        <property name="vexpand">1</property>
                        <layout>
                          <property name="left_attach">1</property>
                        </layout>
                      </object>
                    </child>
                    <child>
                      <object class="GtkLabel" id="time_label">
                        <property name="hexpand">1</property>
                        <property name="vexpand">1</property>
                        <property name="label" translatable="no">00:00.00</property>
                        <attributes>
                          <attribute name="scale" value="10"></attribute>
                          <attribute name="font-features" value="tnum 1"></attribute>
                        </attributes>
                      </object>
                    </child>
                  </object>
                </property>
                <property name="tab">
                  <object class="GtkLabel" id="label2">
                    <property name="label" translatable="no">page 2</property>
                  </object>
                </property>
              </object>
            </child>
            <child>
              <object class="GtkNotebookPage">
                <property name="position">2</property>
                <property name="child">
                  <object class="GtkGrid" id="grid3">
                    <property name="hexpand">1</property>
                    <property name="row_spacing">12</property>
                    <property name="column_spacing">12</property>
                    <child>
                      <object class="GtkImage" id="trophy_image"/>
                    </child>
                    <child>
                      <object class="GtkBox" id="box2">
                        <property name="valign">center</property>
                        <property name="orientation">vertical</property>
                        <child>
                          <object class="GtkLabel" id="result_label">
                            <property name="valign">start</property>
                            <property name="xalign">0</property>
                            <property name="label" translatable="no">Silver Trophy!</property>
                            <property name="use_markup">1</property>
                            <style>
                              <class name="dim-label"/>
                            </style>
                            <attributes>
                              <attribute name="weight" value="bold"></attribute>
                              <attribute name="scale" value="1.5"></attribute>
                            </attributes>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="fill">True</property>
                          </packing>
                        </child>
                        <child>
                          <object class="GtkLabel" id="current_time_label">
                            <property name="valign">start</property>
                            <property name="xalign">0</property>
                            <property name="label" translatable="no">You managed to finish the route with the 2nd best time ever, &lt;b&gt;48.15 seconds!&lt;/b&gt;	</property>
                            <property name="use_markup">1</property>
                            <property name="wrap">1</property>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="fill">True</property>
                          </packing>
                        </child>
                        <child>
                          <object class="GtkLabel" id="better_time_label">
                            <property name="valign">start</property>
                            <property name="xalign">0</property>
                            <property name="label" translatable="no">Only &lt;b&gt;3.75 seconds&lt;/b&gt; separate you from the gold trophy!</property>
                            <property name="use_markup">1</property>
                            <property name="wrap">1</property>
                          </object>
                          <packing>
                            <property name="expand">False</property>
                            <property name="fill">True</property>
                          </packing>
                        </child>
                        <layout>
                          <property name="left_attach">1</property>
                        </layout>
                      </object>
                    </child>
                  </object>
                </property>
                <property name="tab">
                  <object class="GtkLabel" id="label3">
                    <property name="label" translatable="no">page 3</property>
                  </object>
                </property>
              </object>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
          </packing>
        </child>
        <child>
          <object class="GtkButton" id="run_button">
            <property name="can_focus">1</property>
            <property name="receives_default">1</property>
            <property name="hexpand">1</property>
            <property name="vexpand">1</property>
            <property name="use_action_appearance">False</property>
            <child>
              <object class="GtkLabel" id="run_button_label">
                <property name="label" translatable="yes">Run!</property>
                <attributes>
                  <attribute name="scale" value="3"></attribute>
                </attributes>
              </object>
            </child>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
          </packing>
        </child>
      </object>
    </child>
  </object>
</interface>
